react/admin/ReturnForm.tsx

```diff
@@ -736,6 +736,25 @@ class ReturnForm extends Component<any, any> {
     let allowedStatuses: any = [{ label: currentStatus, value: status }]
 
     if (status === requestsStatuses.new) {
+      allowedStatuses.push({
```

Comment:
This looks good, but I think we have an opportunity here to start simplifying the component. How about making a helper function to return `allowedStatuses` in a different file?

Reply:
yeah i am agree with you, so i will do it